src/update-roster-new.R

```diff
@@ -0,0 +1,363 @@
+scrape_old_roster <- function(team, )
+
+build_rosters <-
+  function(season = nflreadr:::most_recent_season(roster = TRUE)) {
+    # if NGS rosters are not available, default to sleeper API
```

Comment:
higher level: is there an nfl data exchange api that we could/should be using instead?

Reply:
ah shouldn't have comitted this. `update-weekly-rosters` uses the data exchange API and does the same thing as this script